File: lua/refactoring/tests/debug/print_var/lua/multiple-statements/print_var.expected.lua
function simple_function(a)
    local test = 1
    local test_other = 11
    -- __AUTO_GENERATED_PRINT_VAR_START__
    print("custom print_var simple_function test_other:", vim.inspect(test_other)) -- __AUTO_GENERATED_PRINT_VAR_END__
    for idx = test - 1, test_other do
        print(idx, a)
    end
end

File: lua/refactoring/tests/debug/print_var/rb/multiple-statements/print_var.expected.rb
class SimpleClass
  def simple_function(a)
    test = 1
    test_other = 11
    # __AUTO_GENERATED_PRINT_VAR_START__
    puts "custom print_var SimpleClass#simple_function test_other: #{test_other}" # __AUTO_GENERATED_PRINT_VAR_END__
    for x in test..test_other do
      puts x, a
    end
  end
end
